src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -247,15 +247,20 @@ public void exportFullpages(String query, Integer startYear, Integer endYear, In
         }
         // Query Solr
         SolrQuery finalQuery = fullpageSolrQuery(query, startYear, endYear, max);
```

Comment:
As both full pages and illustrations are steamed where the `rows` parameter is ignored, `max` should not be set by `fullpageSolrquery`: Although it does not have any effect, the method signature is confusing as it signals that it does.